ui/admin/app/components/form/storage-bucket/aws/index.hbs

```diff
@@ -301,6 +301,7 @@
         @isInvalid={{@model.errors.access_key_id}}
         @isDisabled={{form.disabled}}
         @showEditButton={{true}}
+        @data-test-selector='access_key_id'
```

Comment:
This is [explained here](https://github.com/hashicorp/boundary-ui/pull/2278/files#r1612254959)